doc/specs/stdlib_stats_distribution_uniform.md

```diff
@@ -0,0 +1,381 @@
+---
+title: stats_distribution
```

Comment:
Should this be:

```suggestion
title: stats_distribution_uniform
```

I noticed that `stats_distribution_PRNG` spec has the same `title` value in the header. If this was intended, please ignore this comment, I'm not too familiar with the details of how FORD deals with these things.

Reply:
Yes, the `title` is intended to be `stats_distribution`. The subtitle shows the difference.